packages/office-ui-fabric-react/src/demo/ComponentStatus/ComponentStatusState.tsx

```diff
@@ -0,0 +1,381 @@
+import { IComponentStatusProps, ChecklistStatus } from './ComponentStatus.Props';
+
+export interface IComponentStatusState {
+  [key: string]: IComponentStatusProps;
+}
+
+export const ComponentStatusState: IComponentStatusState = {
+  Breadcrumb: {
```

Comment:
I think these probably should be defined in each component folder next to the page.

Reply:
+1, made the same comment yesterday

Reply:
Yes thanks for the feedback. I'll update the pr to move this to each component. 👍

Reply:
Just updated the PR to reflect this change.